### ISSUE TITLE
'Authorised' in en-US page

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#12276]
With a linked account the text says ‘authorised’ in en-US but should say ‘authorized’:

!nuKUeqWp8pmjrQGej15es0pLFCrQQ9QEqDYqR--PrOHu3W-xKcFchNsML803L1YzOAZs3PiRk60Rs5h1pmXn6idWk8hzpUJ-mvA2hfWN6eUi-qSNDzsiKvxJ_wlomHIqBJ20IlHn|width=1304,height=480!

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-4772)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/22239">nuKUeqWp8pmjrQGej15es0pLFCrQQ9QEqDYqR--PrOHu3W-xKcFchNsML803L1YzOAZs3PiRk60Rs5h1pmXn6idWk8hzpUJ-mvA2hfWN6eUi-qSNDzsiKvxJ_wlomHIqBJ20IlHn</a>

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

Adds “relates to” for fxa-4728 and fxa-4786, this is pretty odd. Three locations with British English language in en-us experiences. Is there something bigger going on here?

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Thanks for the investigation Dan Schomburg .  We all talked on Slack.  I think it makes sense to deploy all the locales we offer including en-US.

**Comment by data-sync-user** [comment]
➤ Dan Schomburg commented:

Out of this discussion also came the notion of reusing our logic in fxa-shared. I took a stab at this, and it seems to be effective, as it addresses a quite a few edge cases surrounding en-US as the ‘default’ language. It does not however address the discrepancy between local development and stage/prod.  

I also reached out to an l10n team member on the fxa channel for further guidance. If there is a a way our org handles this. Being in alignment with their suggestion is probably the best solution.
